src/jaxsim/api/contact.py

```diff
@@ -190,11 +191,11 @@ def in_contact(
 
     links_in_contact = jax.vmap(
         lambda link_index: jnp.where(
-            model.kin_dyn_parameters.contact_parameters.body == link_index,
+            model.physics_model.gc.body == link_index,
             below_terrain,
             jnp.zeros_like(below_terrain, dtype=bool),
         ).any()
-    )(jnp.arange(model.number_of_links()))
+    )(Link.names_to_idxs(link_names=link_names, model=model))
```

Comment:
```suggestion
    )(js.link.names_to_idxs(link_names=link_names, model=model))
```

Reply:
Addressed in https://github.com/ami-iit/jaxsim/pull/119/commits/dd0d00b4c76315b48a2a096157ff6531e5e14037